doc/cookbook/cookbook_snippets.cpp

```diff
@@ -0,0 +1,15 @@
+#include <seqan3/core/debug_stream.hpp>
+
+int main()
+{
+
+    {
+        //![snippet1]
+        //![snippet1]
+    }
```

Comment:
What's this snippet showing? :thinking:

Reply:
Nothing yet, I included it to make it a bit easier for everyone to add their snippets

Reply:
Hm but it is not included anywhere and there is no mention that this serves as a blueprint?
I'm not sure if this may then rather confuse people, like "should I change this or copy this?".
Maybe the other snippets already included here are examples enough?

Reply:
Fair point. I deleted the file.